packages/calcite-design-tokens/support/token-transformer/helpers/createCalciteTokenFiles.spec.ts

```diff
@@ -1,16 +1,20 @@
-import { resolve } from "path";
+import { dirname, resolve } from "path";
+import { fileURLToPath } from "url";
 import { createCalciteTokenFiles } from "./createCalciteTokenFiles.js";
 import { ConfigOptions } from "../../types/config.js";
 
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
 describe("Helper to create a Calcite Token Transformer configuration file", () => {
   it("should return a token file", async () => {
     const args = {
       name: "testName",
-      path: resolve(__dirname, "../../test/mock/src/global.json"),
+      path: resolve(__dirname, "..", "..", "test", "mock", "src", "global.json"),
```

Comment:
@alisonailea when you have some free time, can you go through the design tokens code and do this for all of the `resolve` calls?